src/containers/User/UserProfileLayout.tsx

```diff
@@ -22,12 +22,14 @@ const TabWrapper = ({ children, ...props }: TabWrapperProps) => (
 
 interface Props {
   user: User
-  activeTab: 'creations' | 'articles' | 'collection' | 'on-sale' | 'dashboard'
+  activeTab: 'creations' | 'articles' | 'collection' | 'on-sale' | 'dashboard',
+  hideSectionSpacing?: boolean
```

Comment:
Added this boolean for hiding the spacing above the section content. This was needed to move the `subtabs-vertical` closer to the main tab navigation

Reply:
LGTM